locales-lt-LT.xml

```diff
@@ -258,13 +257,13 @@
 
     <!-- VERB ROLE FORMS -->
     <term name="container-author" form="verb"></term>
-    <term name="director" form="verb">directed by</term>
+    <term name="director" form="verb">vadovavo</term>
```

Comment:
I mean the verb form below (line 271), which (on this pull request) still says "dir.". Illustrator says "illus."